### ISSUE TITLE
[1.7.10] Edit the Quests in Multiplayer doesn't work well

### ISSUE THREAD

**Issue by EverNife** [Funwayguy/BetterQuesting#136]
I have tested a few things and some of them dont work well.

First i created a task inside a quest, ok, it worked, then i tried to delete it... it didn't work

Second i tried to edit a task and change the amout of items needed (i tried Retrieval Task) and change the amout to a number, i put 1000 

For me it shows that worked, but i asked a player to test... he said it was " 1 "
![image](https://cloud.githubusercontent.com/assets/11357801/22356752/d3c4b3ea-e41a-11e6-9274-0f8e4962311c.png)

Then i used `/bq_admin reset all EverNife`

and when i looked again, it was:

![image](https://cloud.githubusercontent.com/assets/11357801/22356771/f26a4eb8-e41a-11e6-9d9f-44e69dac0ae6.png)

I take a look in the json file and it was this way: (Even before using the `/bq_admin reset all EverNife`)

![image](https://cloud.githubusercontent.com/assets/11357801/22356811/2fc651f8-e41b-11e6-8fae-f1850abc62bb.png)

The point is, before the first edit it was 5 dirts... so at the end it edited the amout, but to 1

```
Versions:

BetterQuesting-2.0.208
StandardExpansion-2.0.121
```

**Comment by Funwayguy** [comment]
This appears to be a sync problem with the item selection editor. Any other editors you know of that has this problem?

**Comment by EverNife** [comment]
I tested the rewards too.
None of them sync, all was set to " 1 "
The "choice reward" does not even show the items that were set.

The "Design" seems to work, as well as the "requirements" of quests

That's all that i have observed.

**Comment by Funwayguy** [comment]
Seems task and reward edits didn't make any attempt to sync properly. Just fixed this up in builds 2.0.210 and 2.3.11

**Comment by Funwayguy** [comment]
I'm going to close this as I'm pretty sure the fix for the referenced issue also fixes this.
